package.json

```diff
@@ -23,7 +23,7 @@
     "nodeunit-x": "0.15.0"
   },
   "dependencies": {
-    "re2": "1.20.7"
+    "re2": "1.21.4"
```

Comment:
Also, I was wondering... Why not make this version constraint more relaxed?

```suggestion
    "re2": "^1.21.4"
```

Or is the concern here that newer minor versions of `re2` might break this package?